### ISSUE TITLE
Reproducing modification demo

### ISSUE THREAD

**Issue by WangkiShinran** [zjy526223908/TIP-Editor#13]
Thanks for your excellent work and I have already reproduced the demo of adding a sunglasses to the toy successfully! I also tried some addition to my own data and found the results good.
While I tried to reproduce the modification demo, I found that the original structure will also appear in the final result. For example, I tried to change the lid of a bottle to a blue one. Here is my result:

https://github.com/user-attachments/assets/5d5ae518-1d55-4995-b919-cbe5341a655c
https://github.com/user-attachments/assets/ce9d2ed1-df30-4b5f-8b01-dafea3d3b79d
https://github.com/user-attachments/assets/d3855502-78de-4e1f-9e58-8a492c66486b
https://github.com/user-attachments/assets/a3826f75-1f62-4157-aa75-47302c09f26d

The input image is 

![Image](https://github.com/user-attachments/assets/1ac7f6b3-d7a8-4d96-9218-ed324375ae16)

I am sure about the 3D box I gave is correct. How can I change the parameters to get the lid totally changed?
My sh file is shown as follow:
# train initial 3DGS, we provide trained 3dgs in https://drive.google.com/drive/folders/1DWcMyx5ab7PW3QkCwtJmkKUBL_feQxNW?usp=sharing
python train_3DGS.py --fp16 --workspace ./res_gaussion/bottle  \
--min_opacity 0.001 \
--percent_dense 0.1 \
--iters 80000 \
--data_path /data0/TIP-Editor-main/data/bottle  \
--R_path ./data/bottle/Orient_R.npy  \
--initial_points  ./data/bottle/sparse_points.ply \
--train_resolution_level 2 --eval_resolution_level 2 \
--data_type 'colmap'  --eval_interval 50

# sample views for stepwise personalization
python train_3DGS.py --fp16 --workspace ./res_gaussion/bottle9  \
--test --sample \
--bounding_box_path './data/3d_box/bottle_lid2.ply' \
--radius_list 5  --fovy 50 --phi_list 45 60 75 90 105 120 135  --theta_list 30 45 60

# run stepwise personalization
# 1. scene personalization
export HF_ENDPOINT=https://hf-mirror.com
export MODEL_NAME="stabilityai/stable-diffusion-2-1-base"
export OUTPUT_DIR="./res_gaussion/bottle9/scene_personalization"
export image_root=./res_gaussion/bottle9/sample_views/rgb
python personalization/scene_personalization.py \
  --pretrained_model_name_or_path=$MODEL_NAME  \
  --enable_xformers_memory_efficient_attention \
  --instance_data_dir $image_root \
  --instance_prompt 'a photo of a <ktn> thermos mug' \
  --instance_prompt 'a photo of a <ktn> thermos mug with a blue lid' \
  --with_prior_preservation \
  --class_data_dir './res_gaussion/bottle9/class_samples' \
  --class_prompt 'a photo of a thermos mug' \
  --validation_prompt='a photo of a <ktn> thermos mug with a blue lid' \
  --instance_pos -1 \
  --output_dir $OUTPUT_DIR \
  --validation_images $image_root/5.0_45_75.png \
    $image_root/5.0_45_90.png  \
    $image_root/5.0_45_105.png  \
  --max_train_steps=2000

# 2. novel content personalization
export MODEL_NAME="./res_gaussion/bottle9/scene_personalization/checkpoint-2000"
export OUTPUT_DIR="./res_gaussion/bottle9/content_personalization"

python personalization/content_personalization.py \
--pretrained_model_name_or_path $MODEL_NAME  \
--enable_xformers_memory_efficient_attention \
--instance_data_dir $image_root \
--instance_data_dir "./data/object/lid2" \
--class_data_dir './res_gaussion/bottle9/class_samples' \
--instance_prompt 'a photo of a <ktn> thermos mug' \
--instance_prompt 'a photo of a <pth> blue lid' \
--class_prompt 'a photo of a thermos mug' \
--validation_prompt "a photo of a <ktn> thermos mug with a <pth> blue lid"  \
--output_dir $OUTPUT_DIR \
--scene_frequency 200 \
--validation_images $image_root/5.0_45_75.png \
  $image_root/5.0_45_90.png  \
  $image_root/5.0_45_105.png  \
--max_train_steps=1000

# 3. Coarse Editing via SDS Loss
python train_coarse_editing.py  \
  --iters 10000 \
  --reset_points True \
  --editing_type 0 \
  --bbox_size_factor 1.25 \
  --batch_size 2 \
  --seed 1 --eval_interval 4   \
  --load_path  ./res_gaussion/bottle9/checkpoints/df_ep0651.pth \
  --bbox_path ./data/3d_box/bottle_lid2.ply \
  --text_global "a photo of a <ktn> thermos mug with a <pth> blue lid"    	\
  --text_local "a photo of a <pth> blue lid"      	\
  --sd_path ./res_gaussion/bottle9/content_personalization/checkpoint-1000  \
  --radius_range 5.0 5.0 --fovy_range 50 50 --pose_sample_strategy '360' \
  --phi_range 45 135 --theta_range 30 60   \
  --workspace ./res_gaussion/bottle9/coarse_editing_res

# 4. Pixel-Level Image Refinement
python train_refinement.py  \
  --eval_interval 20  \
  --iters 1000 \
  --seed 1   \
  --coarse_gs_path res_gaussion/bottle9/coarse_editing_res/checkpoints/None.pth \
  --initial_gs_path res_gaussion/bottle9/checkpoints/df_ep0651.pth  \
  --bbox_path ./data/3d_box/bottle_lid2.ply \
  --text_global "a photo of a <ktn> thermos mug with a <pth> blue lid" \
  --sd_path ./res_gaussion/bottle9/content_personalization/checkpoint-1000  \
  --radius_range 5.0 5.0 --fovy_range 50 50 --pose_sample_strategy '360' \
  --phi_range 0 90 --theta_range 45 45 \
  --radius_list 5.0 \
  --phi_list 45 60 75 90 105 120 135 \
  --theta_list 45 60 \
  --workspace res_gaussion/bottle9/refine_res/

Thanks!
